RESTserver/src/main/java/com/softserve/if072/restservice/service/StorageService.java

```diff
@@ -27,7 +27,7 @@
     private final StorageDAO storageDAO;
     private final ShoppingListService shoppingListService;
     private final HistoryService historyService;
-    private final int limit = 1;
+    private static final int limit = 1;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00115)